packages/javalib/javalib.2.3.2/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "1.2"
+maintainer: "sawja@inria.fr"
+build: [
+  ["./configure.sh"]
+  [make "ptrees"]
+  [make "installptrees"]
+  [make]
+]
+bug-reports: "sawja@inria.fr"
+install: [
+  [make "install"]
+]
+remove: [
+  ["ocamlfind" "remove" "javalib"]
+  ["ocamlfind" "remove" "ptrees"]
+]
+homepage: "http://sawja.inria.fr"
+depends: [
+  "ocamlfind"
+  "camlzip" {= "1.05"}
+  "camlp4" {>= "4.03"}
```

Comment:
This dependency is painful indeed.
Solution 1 : remove it. But we use stream in jParseSignature.ml for example. Do you know a simple workaround ?
Solution 2 : explain that camlp4-version = ocaml-version in some way ?